dev/reference/tools/tidb-binlog/operation.md

```diff
@@ -127,3 +128,74 @@ Command example:
     ```
 
     This command generates a `{data-dir}/savepoint` file. This file stores the `tso` information which is needed for the initial start of Drainer.
+
+## Use SQL statements to control Pump or Drainer
+
+<<<<<<< HEAD
+Execute the corresponded SQL statements in the TiDB server to check or modify the states of binlog.
+=======
+To view or modify binlog related states, execute corresponding SQL statements in TiDB.
+>>>>>>> 36c13cff2696e78ee84f08e0b4c286859d5405fd
+
+- Check whether binlog is enabled:
+
+    ```bash
+    mysql> show variables like "log_bin";
+    +---------------+-------+
+    | Variable_name | Value |
+    +---------------+-------+
+    | log_bin       |  ON   |
+    +---------------+-------+
+    ```
+
+    When the Value is `ON`, it means that the binlog is enabled.
+
+- Check the status of all the Pump or Drainer nodes:
+
+    ```bash
+    mysql> show pump status;
+    +--------|----------------|--------|--------------------|---------------------|
+    | NodeID |     Address    | State  |   Max_Commit_Ts    |    Update_Time      |
+    +--------|----------------|--------|--------------------|---------------------|
+    | pump1  | 127.0.0.1:8250 | Online | 408553768673342237 | 2019-05-01 00:00:01 |
+    +--------|----------------|--------|--------------------|---------------------|
+    | pump2  | 127.0.0.2:8250 | Online | 408553768673342335 | 2019-05-01 00:00:02 |
+    +--------|----------------|--------|--------------------|---------------------|
+    ```
+
+    ```bash
+    mysql> show drainer status;
+    +----------|----------------|--------|--------------------|---------------------|
+    |  NodeID  |     Address    | State  |   Max_Commit_Ts    |    Update_Time      |
+    +----------|----------------|--------|--------------------|---------------------|
+    | drainer1 | 127.0.0.3:8249 | Online | 408553768673342532 | 2019-05-01 00:00:03 |
+    +----------|----------------|--------|--------------------|---------------------|
+    | drainer2 | 127.0.0.4:8249 | Online | 408553768673345531 | 2019-05-01 00:00:04 |
+    +----------|----------------|--------|--------------------|---------------------|
+    ```
+
+<<<<<<< HEAD
+- Modify the states of the Pump or Drainer node:
+=======
+- Modify the state of a Pump or Drainer node:
+>>>>>>> 36c13cff2696e78ee84f08e0b4c286859d5405fd
+
+    ```bach
+    mysql> change pump to node_state ='paused' for node_id 'pump1'";
+    Query OK, 0 rows affected (0.01 sec)
+    ```
+
+    ```bach
+    mysql> change drainer to node_state ='paused' for node_id 'drainer1'";
+    Query OK, 0 rows affected (0.01 sec)
+    ```
+
+> **Note:**
+>
+<<<<<<< HEAD
```

Comment:
It looks like you are accidentally committing a git merge here.

Reply:
It seems the conflict arises when I commit the suggestion on the github web while some commits are still in the desktop. Thanks for your reminder!